run.sh

```diff
@@ -0,0 +1,111 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage:
+#   sh run.sh lint path/to/config.jsonc path/to/package
+#   sh run.sh test path/to/config.jsonc path/to/package
+
+# For debugging:
+#   sh -x run.sh lint path/to/config.jsonc path/to/package
+#   sh -x run.sh test path/to/config.jsonc path/to/package
+
+set -e # Exit immediately if a command exits with a non-zero status.
+
+if [ -z "$PROJECT_ID"]; then
+  echo "The PROJECT_ID environment variable is required." >&2
+  exit 1
+fi
+
+set -o nounset # Exit immediately if a variable is used but not set.
+command="$1"
+config_file="$2"
+package_path="$3"
+
+# Print versions to make sure dependencies are installed.
+jq --version
+gcloud --version
```

Comment:
For some reason on my system this command take 3.5s. Unsure if it's the same on yours; would a `which gcloud` be a valid alternative here?

Reply:
Good point, yes that would work